app/models/user.rb

```diff
@@ -11,12 +11,28 @@ class User < RedisStore
   attribute :last_name
   attribute :zip
 
+  # vaafi attributes
+  attribute :edipi
+  attribute :issue_instant
+  attribute :participant_id
+  attribute :ssn
```

Comment:
This is probably more of a question for @saneshark but is there a reason to use Virtus attribute definitions here instead of just Ruby `attr_accessor`? I don't see any coercion happening but can also understand about wanting to be consistent if some attributes are being defined with Virtus already